javascript/chronometer.js

```diff
@@ -1,26 +1,39 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime =0;
+    this.intervalId=0;
+    this.intervalId2=0;
+    this.currentMilli=0;
   }
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(()=>{this.currentTime++}, 1000)
+    this.intervalId2 = setInterval(()=>{this.currentMilli++}, 10)
+    console.log ('started')
   }
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime/60)
   }
   getSeconds() {
-    // ... your code goes here
+    return Math.floor(this.currentTime%60)
   }
-  twoDigitsNumber() {
-    // ... your code goes here
+  getMilliseconds() {
+    return Math.floor(this.currentMilli%100)
+  }
+
+  twoDigitsNumber(number) {
+    if (number>=10){return `${number}`}
```

Comment:
Just a little tip. When doing one line conditionals, you can remove the brackets and it will look better and still easy to read. Another tip would be for you to turn this into a ternary and start to get comfortable with those.